Content.Server/Atmos/Atmospherics.cs

```diff
@@ -0,0 +1,146 @@
+﻿using Robust.Shared.IoC;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.Atmos
+{
+    /// <summary>
+    ///     Class to store atmos constants.
+    /// </summary>
+    public static class Atmospherics
+    {
+        public static GasPrototype GetGas(int gasId) =>
+            IoCManager.Resolve<IPrototypeManager>().Index<GasPrototype>(gasId.ToString());
```

Comment:
As stated before, this can be optimized significantly and will probably be on hot paths too much for `IoCManager.Resolve<>` to not be a bottleneck.